specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/modelManagement.json

```diff
@@ -0,0 +1,4755 @@
+          },
+          {
+            "name": "resourceGroup",
+            "in": "path",
+            "description": "The name of resource group in which the workspace is located.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "workspace",
+            "in": "path",
+            "description": "The name of the workspace.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "202": {
+            "description": "The request was accepted.  The header 'Operation-Location' contains the async operation location URL.  Accessing this URL with a GET call will return the status of the background task."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ModelErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/modelmanagement/v{version}/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.MachineLearningServices/workspaces/{workspace}/services": {
+      "get": {
+        "tags": [
+          "Service"
+        ],
+        "summary": "Query the list of Services in a Workspace.",
+        "description": "If no filter is passed, the query lists all Services in the Workspace. The returned list is paginated and the count of item in each page is an optional parameter.",
+        "operationId": "Services_ListQuery",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "imageId",
+            "in": "query",
+            "description": "The Image Id.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "imageName",
+            "in": "query",
+            "description": "The Image name.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "modelId",
+            "in": "query",
+            "description": "The Model Id.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "modelName",
+            "in": "query",
+            "description": "The Model name.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "name",
+            "in": "query",
+            "description": "The object name.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "tag",
+            "in": "query",
+            "description": "The object tag.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "count",
+            "in": "query",
+            "descript
```

Comment:
api-version is in the query params and version is in the path. Both are described as "The version of the Microsoft.MachineLearning resource provider API to use."

Which one is correct? Are they both needed??

Reply:
Fix on the way for Modelmanagement service APIs to make the endpoints reflect similar structure as the other swaggers